_episodes/03-github-pages.md

```diff
@@ -1,21 +1,152 @@
+- "GitHub serves pages generated from Markdown (and other) files on a branch of a GitHub repository specified by the user"
+- "By convention, if you create a branch called `gh-pages` it will automatically be published as a website by GitHub
+on a special GitHub IO URL composed from your GitHub name and repository name"
 ---
 
-- push to a GitHub repo
-- activate Pages in the repo settings
-- change something in `index.md`
-- go to landing page
-- make a change in `index.md` and see how landing page updates
-- add a second `.md`
-- add minimal YAML header
+# What is GitHub Pages?
+[GitHub Pages](https://docs.github.com/en/github/working-with-github-pages/about-github-pages) is a free website 
+hosting service by GitHub that takes files from your GitHub repository that is configured as a website, 
+optionally runs the files through a build process, and publishes them as a website or a web page. 
+Any changes you do to the files in your website's GitHub repository 
+will be rendered live in the website.
+
+[Anne's bit]
+
+# Website Configuration
+So far we have seen how to configure your repository to host a website and where to find the rendered website. Let's
+now look into the configuration of the website itself.
+
+## Configuring Parameters via `_config.yml`
+We have already mentioned that GitHub has an engine under the hood to process your files and 
+build functional websites out of your content. 
+This engine is called Jekyll and we will talk more about Jekyll in the following episodes. 
+Jekyll's main configuration options are specified in `_config.yml` file placed in your site’s root directory. 
+`_config.yml` file is written 
+in the language called [YAML](https://yaml.org/) - a human-readable data-serialization language that allows us to define scalar 
+parameters (that contain single values) as well as parameters that contain lists and dictionaries (hashes). 
+
+> ## YAML 
+>
+> [YAML](https://yaml.org/) is a human-readable data-serialization language. It is commonly used for configuration files and in 
+> applications where text data is being stored or transmitted and it is programming language agnostic.  
+>
+{: .callout}
+
+Let's create some configuration parameters for our website.
+
+1. From the GitHub interface, create `_config.yml` file in your site’s root directory.
+2. Add parameters `title`, `description` and `email` to it as:
+
+    ~~~
+    title: "Research Project Website"
+    description: "Research Project Description"
+    email: team@my.research.org
+    ~~~  
+    {: .language-yaml}
+
+3. Commit your changes.
+
+Site-wide information and configuration settings from 
+`_config.yml` are made available as `site.PARAMETER_NAME` variable in every page/file within the website/repository. 
+There is a number of 
+[predefined site-wide variables](https://jekyllrb.com/docs/variables#site-variables) available to you 
+(such as `site.title` and `site.email`). Others (such as `site.description`) you can define yourself. 
```

Comment:
Are `site.title` and `site.email` predefined? Could you choose some different examples here, as it's potentially confusing to talk about these being predefined vs `site.description`, when all three are defined by the user in the steps described above.

Reply:
Hmm, site.title seems to be predefined to the name of the repo. I think I made a mistake about site.email. I can just ignore all that and mention what users can define.

Reply:
It is probably worth explaing site.title. My guess is that it would be the first thing one wants to update. Then it also changes the top of your website (default is the name of your repo and then it will become your title).

Reply:
OK, I've fixed the predefined and custom parameters a bit, see e7f6bb9.

Reply:
I've also fixed the {. :source} bit for Markdown in the same commit e7f6bb9 but I expect that to change as we need a general decision on this in order to be consistent across the materials.